README.md

```diff
@@ -1,29 +1,31 @@
 ![Logo](web/public/favicon.svg)
 
-# JLC PCB SMD Assembly Component Catalogue
+# JLCPCB SMD Assembly Component Catalogue
 
-A better tool to browse the components offered by the [JLC PCB SMT Assembly
+A better tool to browse the components offered by the [JLCPCB SMT Assembly
 Service](https://jlcpcb.com/smt-assembly).
 
 ## How To Use It?
 
 Just visit: [https://yaqwsx.github.io/jlcparts/](https://yaqwsx.github.io/jlcparts/)
 
+The site and parts cache is hosted on GitHub Pages.
+
```

Comment:
This comment makes no logical sense here. This is the first information the user (e.g., electrical engineer) sees when using this project. Why should they care about the internal implementation details of the project? This just clutters the text for them.

Reply:
We should factor the whole `## Developer` section out then. As a developer trying to contribute to this project, I had an embarassingly tough time figuring out how to get started.

Reply:
I agree that having a separate dev section is a good direction.